wiki/en/en-Contribution.md

```diff
@@ -7,32 +7,13 @@ permalink: "/wiki/Contribution"
 
 # Contributing to the Jamulus Project
 
-## Jamulus Software
+If you find a mistake, typo or something out of date (in any language) on this website or in Jamulus itself, you can <a href="https://github.com/jamulussoftware/jamulus/issues">raise it as a bug</a>.
 
-If you want to contribute code or have a feature request, [open an issue on GitHub](https://github.com/jamulussoftware/jamulus/issues/) (preferred) or [post your idea on the forum](https://github.com/jamulussoftware/jamulus/discussions) if you don't have a GitHub account. This allows us to discuss the changes before e.g. making a pull request (be sure to read main dev's [Volker's manifesto for contributions on GitHub](https://github.com/jamulussoftware/jamulus/discussions/915) and the [CONTRIBUTING file](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md))
+If you think some documentation or information is missing or can be improved, <a href="https://github.com/jamulussoftware/jamulus/discussions">post about that on this forum</a> so that it can be discussed first.
 
-See the [Jamulus GitHub repo](https://github.com/jamulussoftware/jamulus) for further information.
+If you want to help out with translation, you will need to have a GitHub account and be familiar with using git and Jekyll. Read about our [documentation and translation process](https://github.com/jamulussoftware/jamuluswebsite/blob/changes/CONTRIBUTING.md), and [introduce yourself](https://github.com/jamulussoftware/jamulus/discussions) if you’d like to get involved. 
```

Comment:
Maybe: „should be familiar with git“ we can probably also help people with git?

Reply:
OK sure.